packages/web-client/services/auth.service.ts

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-underscore-dangle */
+import axios from 'axios';
+import jwtDecode from 'jwt-decode';
+
+class AuthService {
+  public user: null;
+
+  private static _instance: AuthService;
+
+  public static get instance(): AuthService {
+    if (this._instance === undefined) {
+      this._instance = new AuthService();
+    }
+    return this._instance;
+  }
+
+  async login(payload: any): Promise<any> {
+    try {
+      const res = await axios.post('http://localhost:3000/api/v1/auth/login', payload);
```

Comment:
Remove the host section of the url, you can either make a proxy in the package.json, or set a default host in the axios config.